src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsIds;
+    private Long userId;
```

Comment:
I think you don't need to return id of shopping cart, because it equals to id of user, just return userId and ticketsIds

Reply:
Basically, it is better to return the id of the entity in all cases. This is a project-specific case when the user id and shopping cart id are the same